src/routes/components/components.json

```diff
@@ -2981,5 +2981,210 @@
 		"category": "Data Visualisation",
 		"tags": ["components and libraries", "images"],
 		"stars": 0
+	},
+	{
+		"title": "svelte-transition",
+		"url": "https://captaincodeman.github.io/svelte-headlessui/transition/",
+		"description": "Svelte component to apply CSS classes matching the syntax in TailwindUI components",
+		"npm": "svelte-transition",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": [
+			"components and libraries",
+			"animations",
+			"typescript"
+		],
+		"stars": 35
```

Comment:
The `stars` field is not used anymore in this file.
The count should be provided here: `src/lib/stars.json`

_(This comment is applicable for all components in this PR)_
```suggestion
```